src/vma/util/sys_vars.cpp

```diff
@@ -336,6 +336,41 @@ void mce_sys_var::read_env_variable_with_pid(char* mce_sys_name, size_t mce_sys_
 	}
 }
 
+bool mce_sys_var::check_cpuinfo_flag(const char* flag)
```

Comment:
The is_hypervisor flag is used to fix the illegal instruction error in the instruction movntdq xmm0, register to the BlueFlame page pointed by the AX register. Please note the COPY_64B works fine in KVM and the problem is only accessing the BF page. The is_hypervisor flag is used for alternative implementation of the COPY_64B in VM. The flag also can be used to fix specific bugs for VM as well.